pkgs/build-support/build-fhsenv-bubblewrap/buildFHSEnv.nix

```diff
@@ -76,6 +76,12 @@ let
     # XDG_DATA_DIRS is used by pressure-vessel (steam proton) and vulkan loaders to find the corresponding icd
     export XDG_DATA_DIRS=$XDG_DATA_DIRS''${XDG_DATA_DIRS:+:}/run/opengl-driver/share:/run/opengl-driver-32/share
 
+    # Following XDG spec [1], XDG_DATA_DIRS should default to "/usr/local/share:/usr/share".
+    # In nix, it is commonly set without containing these values, so we add them as fallback.
+    #
+    # [1] <https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html>
+    export XDG_DATA_DIRS=$XDG_DATA_DIRS''${XDG_DATA_DIRS:+:}/usr/local/share:/usr/share
```

Comment:
All those are missing the makeWrapper dedupe logic.

Reply:
I don't understand how I could import that logic here without code duplication, since we don't have an executable, and `addValue()` is private.
Also, if what we are trying to achieve is "Add fallback values if they don't exist", we cannot use `addValue()` because it may change the order of values:

```sh
$ export XDG_DATA_DIRS="/usr/share:$HOME/my-data-dir-with-buggy-stuff"
$ steam-run bash
$ echo $XDG_DATA_DIRS
"$HOME/my-data-dir-with-buggy-stuff:/usr/share"
```

Maybe something like this would not be too bad to include inline?

```sh
case ":$XDG_DATA_DIRS:" in
  *:/usr/share:*) ;;
  *) export XDG_DATA_DIRS="$XDG_DATA_DIRS''${XDG_DATA_DIRS:+:}/usr/share" ;;
esac
```

Reply:
I think we would need to duplicate code here. Also probably something for another PR.